.github/actions/setup-backend/action.yml

```diff
@@ -29,6 +29,8 @@ runs:
           echo "PYTHON_VERSION=3.10" >> $GITHUB_ENV
         elif [ "${{ inputs.python-version }}" = "next" ]; then
           echo "PYTHON_VERSION=3.11" >> $GITHUB_ENV
+        elif [ "${{ inputs.python-version }}" = "previous" ]; then
```

Comment:
@mistercrunch I wonder if `previous`, `current`, and `next` are somewhat abstract, i.e., what does "previous" actually mean? Would it be simpler to just use the version name?